tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -41,20 +41,73 @@ This exercise assumes that:
 2. Your Protobuf definition files are in `./proto/`.
 3. You want to transpile them into TypeScript in `./src/codegen/`.
 
-Install `telescope` under your package:
+Install `telescope`, in your package or in docker:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
 
 ```sh
 $ npm install --save-dev --save-exact @cosmology/telescope@1.0.4
 ```
 
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ docker run --rm -it \
+    -v $(pwd):/myLib \
+    -w /myLib \
+    checkers_i \
+    npm install --save-dev --save-exact @cosmology/telescope@1.0.4
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="DockerGlobal">
+
+```sh
+# Install Node
+RUN curl -fsSL https://deb.nodesource.com/setup_${NODE_VERSION} | bash -
+RUN apt-get install -y nodejs
```

Comment:
What about:

```
FROM node:x.y
```

Like this you also make explicit which Node version will work.